proto/cosmos/group/module/v1/module.proto

```diff
@@ -18,7 +18,21 @@ message Module {
   google.protobuf.Duration max_execution_period = 1
       [(gogoproto.stdduration) = true, (gogoproto.nullable) = false, (amino.dont_omitempty) = true];
 
-  // max_metadata_len defines the max length of the metadata bytes field for various entities within the group module.
-  // Defaults to 255 if not explicitly set.
+  // MaxMetadataLen defines the max chars allowed in all
+  // messages that allows creating or updating a group
+  // with a metadata field
+  // Defaults to 140 if not explicitly set.
   uint64 max_metadata_len = 2;
+
+  // MaxProposalTitleLen defines the max chars allowed
+  // in string for the MsgSubmitProposal and Proposal
+  // summary field
+  // Defaults to 140 if not explicitly set.
```

Comment:
Ditto

Reply:
Thanks for pointing that out, consequences of copy paste